### PR TITLE
EDU-3272: Rewords and clarifies how to ask for help

### PR DESCRIPTION
**Pull request by fairlydurable**
For: self-hosted Multi-Cluster Replication

Ask: 

> The docs to set up multi-cluster replication for self-hosted clusters says "Please reach out to us if you need to se this up". It is not clear what this means. Should they make a request in one of the community slack channels? Or something else?
> Proposed solution:
> https://docs.temporal.io/self-hosted-guide/multi-cluster-replication#set-up-multi-cluster-replication-prior-to-v114
> Recommend a link to Community Slack channel at a minimum (or remove this)

### CODE REVIEW

File: docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx [hunk 1]
```diff
@@ -428,7 +428,16 @@ After the above conditions are satisfied, you can start to configure a multi-clu
 #### Set up Multi-Cluster Replication prior to v1.14
 
 You can set this up with [`clusterMetadata` configuration](/references/configuration#clustermetadata); however, this is meant to be only a conceptual guide rather than a detailed tutorial.
-Please reach out to us if you need to set this up.
+
+:::tip
+
+If you need help when setting up, please reach out to our [community Slack](https://temporalio.slack.com).
+Good places to start include searching through previous conversations and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
```
Comment (angelazhou32) [inline docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx:435]: How about suggesting a channel? Reaching out to the community Slack might be a bit vague. Maybe the #support-community would be a good suggestion.

File: docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx [hunk 2]
```diff
@@ -428,7 +428,16 @@ After the above conditions are satisfied, you can start to configure a multi-clu
...
+
+Need a Slack invitation?
+Here's a [invitation link](https://temporal.io/slack).
```
[the remark this answers is not in the document]
Comment (fairlydurable) [reply docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx:438]: Hi @angelazhou32 

The link is different, and you can test that by using a private browser. Correcting the typo.

File: docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx (hunk 1 above)
Comment (fairlydurable) [inline docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx:435]:
```suggestion
Good places to start include the **\#support-community** channel, searching through previous conversations and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
```

File: docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx [hunk 3]
```diff
@@ -428,7 +428,16 @@ After the above conditions are satisfied, you can start to configure a multi-clu
...
 You can set this up with [`clusterMetadata` configuration](/references/configuration#clustermetadata); however, this is meant to be only a conceptual guide rather than a detailed tutorial.
-Please reach out to us if you need to set this up.
+
+:::tip
+
+If you need help when setting up, please reach out to our [community Slack](https://temporalio.slack.com).
+Good places to start include the **\#support-community** channel, searching through previous conversations and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
+
+Need a Slack invitation?
+Here's a [invitation link](https://temporal.io/slack).
```
Comment (fairlydurable) [inline docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx:438]:
```suggestion
Here's an [invitation link](https://temporal.io/slack).
```

**Review by angelazhou32 — APPROVED** [review]
LGTM!

File: docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx [hunk 4]
```diff
@@ -428,7 +428,16 @@ After the above conditions are satisfied, you can start to configure a multi-clu
 #### Set up Multi-Cluster Replication prior to v1.14
 
 You can set this up with [`clusterMetadata` configuration](/references/configuration#clustermetadata); however, this is meant to be only a conceptual guide rather than a detailed tutorial.
-Please reach out to us if you need to set this up.
+
+:::tip
+
+If you need help when setting up, please reach out to our [community Slack](https://temporalio.slack.com).
+Good places to start include the **\#support-community** channel, searching through previous conversations and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
```
Comment (fairlydurable) [inline docs/production-deployment/self-hosted-guide/multi-cluster-replication.mdx:435]:
```suggestion
Good places to start include the **\#support-community** channel, searching through previous conversations, and asking our unusually excellent Temporal-trained large language model (visit **\#ask-ai**).
```

**Pull request merged**